shapely/geometry/base.py

```diff
@@ -181,14 +204,23 @@ def svg(self, scale_factor=1.0, **kwargs):
         """Raises NotImplementedError"""
         raise NotImplementedError
 
+    def svg_path_element(self, stroke_scale_factor=1.0):
+        path = ET.fromstring(self.svg(stroke_scale_factor))
+
+        # svg is y-down, shapely is y-up
+        _, ymin, _, ymax = self.bounds
+        path.set('transform', f'matrix(1,0,0,-1,0,{ymax + ymin})')
+        return path
+
+    def svg_doc(self, width, height, xmin, ymin, xmax, ymax, stroke_scale_factor=1):
+        path = self.svg_path_element(stroke_scale_factor)
+        doc_element = svg_doc_element([path], width, height, xmin, ymin, xmax, ymax)
+        return ET.tostring(doc_element, encoding="unicode")
```

Comment:
How about eliminating this function and where it is called, substitute `svg_doc_element([self.svg_path_element(stroke_scale_factor)], width, height, xmin, ymin, xmax, ymax)` ?

Reply:
With the elimination of `svg_doc()`, the user needs to figure out that they need to set the encoding to "unicode" or the SVG won't be valid. That wasn't immediately obvious to me, but I was able to figure it out after brushing up on ElementTrees. It seems reasonable to make this compromise, as the answer is already in the Python docs.